CHANGELOG.md

```diff
@@ -1,4 +1 @@
 ## 1.71.2 (2023-02-01)
-
-
-
```

Comment:
These extra lines must be added in order for the file to not be modified.

Reply:
Thanks